class-tgm-plugin-activation.php

```diff
@@ -451,6 +451,8 @@ public function init() {
 			if ( true === $this->has_forced_deactivation ) {
 				add_action( 'switch_theme', array( $this, 'force_deactivation' ) );
 			}
+
+			add_action( 'admin_head',  array( $this, 'admin_css' ) );
```

Comment:
Please add a comment above this line to indicate why this hook is there - something along the lines of `// Add CSS to for the TGMPA admin page.`